README.md

```diff
@@ -30,6 +29,17 @@ AWS will apply the following tags when resources are provisioned with the Servic
 This custom resource uses these tags to retrieve more information and applies
 them as additional tags on the provisioned resource.
 
+### Supported Tags
+
+#### All Resources
+* Synapse tags - Retrieve the [Synapse userProfile](https://docs.synapse.org/rest/org/sagebionetworks/repo/model/UserProfile.html)
+info and apply a subset of that data as tags to resources.
+* Marketplace SC product tags - Get Marketplace SC product code and customer ID from subscriber Dynamo DB and
+apply them as tags.
```

Comment:
Can you confirm that this information is not visible to end users?  The concern is that one end user might see the product code and customer ID on another user's instance and then use it to send them a bill.

Reply:
tags are visible to end users.  The SC UI shows tags for the resources that the current end user provisioned.   
Any end user can access tags on resources that other end users provision by going into the EC2 console and viewing them there.

Reply:
I'm concerned that someone who knows a product code and customer id can call the metering API and send someone a bill.  Is there something that prevents this?

Reply:
The alternative to the approach you are taking is to simply use the existing userid tag and to look up the product code and customer id 'on the fly' when it's time to call the metering API. Is there a reason you prefer adding these tags to the resources. (One argument in favor I can think of is that one Synapse user might change their subscription, resulting in different customer ids on different resources for the same user.  Is that what you were thinking of?)

Reply:
> I'm concerned that someone who knows a product code and customer id can call the metering API and send someone a bill. Is there something that prevents this?

Yes, not just any user can call the metering API.  You need marketplace access which we will need to provide to the thing (EC2 or lambda) that will be calling the metering API.

> The alternative to the approach you are taking is to simply use the existing userid tag and to look up the product code and customer id 'on the fly' when it's time to call the metering API. Is there a reason you prefer adding these tags to the resources? (One argument in favor I can think of is that one Synapse user might change their subscription, resulting in different customer ids on different resources for the same Synapse user. Is that what you were thinking of?)

In general i think it's a good practice to tag resources with as much info as possible to provide context about the resource.  I think this will give us greater flexibility to mange changes to those resources.  We can do both your alternative suggestion and tag resources as well, correct?

Reply:
OK